alexandra/app.py

```diff
@@ -114,7 +127,14 @@ def noargs_intent():
 
         # nested decorator so we can have params.
         def _decorator(func):
-            if func.func_code.co_argcount not in [0, 2]:
+            code_list = inspect.getmembers(func, inspect.iscode)
+
+            # Python 2.7 compatibility.
```

Comment:
Should probably break this out in a function, it's a little not DRY since it happens in two places